src/com/sun/ts/tests/webservices13/servlet/WSMTOMFeaturesTestUsingDDs/custom-client.xml

```diff
@@ -18,7 +18,7 @@
 -->
 
 <bindings wsdlLocation="MTOMTestService.wsdl"
-         xmlns="http://java.sun.com/xml/ns/jaxws" 
+         xmlns="https://jakarta.ee/xml/ns/jaxws"
          xmlns:wsdl="http://schemas.xmlsoap.org/wsdl/" 
          xmlns:xsd="http://www.w3.org/2001/XMLSchema" 
          xmlns:jaxb="http://java.sun.com/xml/ns/jaxb">
```

Comment:
missed to update the jaxb namespace here and line 33

Please look for more such.

Reply:
fixed